services/movie.py

```diff
@@ -6,35 +7,40 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
-    queryset = Movie.objects.all()
+    movie = Movie.objects.all()
 
     if genres_ids:
-        queryset = queryset.filter(genres__id__in=genres_ids)
+        movie = movie.filter(genres__id__in=genres_ids)
 
     if actors_ids:
-        queryset = queryset.filter(actors__id__in=actors_ids)
+        movie = movie.filter(actors__id__in=actors_ids)
 
-    return queryset
+    if title:
+        movie = Movie.objects.filter(title__contains=title)
+
+    return movie
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
 def create_movie(
-    movie_title: str,
     movie_description: str,
+    movie_title: str = None,
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Consider specifying the type of elements in the list for `genres_ids` and `actors_ids` in the `create_movie` function. For example, use `list[int]` instead of `list` for clarity and type safety.